_docs/schema/basics.md

```diff
@@ -701,16 +701,20 @@ var options = new EvaluationOptions
 }
 ```
 
+> For languages to become available, you need to add extra dependencies on the specific nuget packages.
+{: .prompt-info }
+
```

Comment:
Please move this note to below the list.

Reply:
Sure. Done. :+1: